src/components/Launch/LaunchForm/StructInput.tsx

```diff
@@ -21,6 +21,25 @@ muiTheme.typography.h5 = {
     fontWeight: 400
 };
 
+const formatJson = data => {
+    const keys = Object.keys(data);
+
+    if (keys.includes('title')) {
+        const { title, type, format } = data;
+        data['title'] = `${title} (${format ? format : type})`;
+        if (!keys.includes('additionalProperties')) return data;
```

Comment:
What if we have other fields but don't have `additionalProperties`?
Just to make sure what this means on my end.

Reply:
'additionalProperties' is all I can see from the api response for now. If there is anything more, we can cope with it later.

Reply:
Yeah, we can do that later.